scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -730,20 +730,21 @@
 (defn make-offer-handler
   [conn driver-atom fenzo fid-atom pending-jobs-atom offer-cache
    max-considerable scaleback
-   floor-iterations-before-warn floor-iterations-before-reset]
+   floor-iterations-before-warn floor-iterations-before-reset
+   trigger-chan]
   (let [chan-length 100
         offers-chan (async/chan (async/buffer chan-length))
         resources-atom (atom (view-incubating-offers fenzo))
-        timer-chan (chime-ch (periodic/periodic-seq (time/now) (time/seconds 1))
-                             {:ch (async/chan (async/sliding-buffer 1))})]
-    (async/thread
-      (loop [num-considerable max-considerable]
-        (reset! fenzo-num-considerable-atom num-considerable)
-
+        num-considerable-atom (reset! fenzo-num-considerable-atom max-considerable)]
```

Comment:
Should this `reset!` be outside the `let`? Also, `num-considerable-atom` is probably not an atom.